.yarn/patches/@jupyterlite-contents-npm-0.7.0-alpha.5-2801e8036a.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/lib/drivefs.js b/lib/drivefs.js
+index 5b5569a23e6f1aadd5f9c40da6f2a21306888d54..0bb034d2e34eb067a1206c8944663830d5bfc651 100644
+--- a/lib/drivefs.js
++++ b/lib/drivefs.js
+@@ -234,7 +234,7 @@ export class DriveFSEmscriptenNodeOps {
+             throw new this.fs.FS.ErrnoError(this.fs.ERRNO_CODES['EPERM']);
+         };
+         this.readlink = (node) => {
+-            throw new this.fs.FS.ErrnoError(this.fs.ERRNO_CODES['EPERM']);
++            console.log("XXX", node); throw new this.fs.FS.ErrnoError(this.fs.ERRNO_CODES['EINVAL']);
```

Comment:
This patch has `EINVAL`, while https://github.com/jupyterlite/jupyterlite/pull/1723 uses `EINVAR`?

Can this explain the UI test failure seen in https://github.com/jupyterlite/jupyterlite/pull/1741 ?